doc/sphinx/source/lua_tribes_units.rst.org

```diff
@@ -64,7 +64,7 @@ The matching ``register.lua`` would look like this:
    Also, ``berry_bush_blueberry_small`` in this example has no attributes, so its
    attribute table is empty.
 
-Every unit defined by an add-on needs to define exactly one of the special attributes ``__skip_if_exists`` or ``__replace_if_exists``. This attribute determines how the engine proceeds if a description of the same name has already been registered. Units not contained in an add-on may not use these attributes.
+Additionaly to its normal attributes (:method:`MapObject.has_attribute`), every unit defined by an add-on needs to define exactly one of the special attributes ``__skip_if_exists`` or ``__replace_if_exists``. This attribute determines how the engine proceeds if a description of the same name has already been registered. Units not contained in an add-on may not use these attributes.
```

Comment:
sorry i did not recognize the context in this file in the first case. As the normal attributes asre explained above we could simply skip the reference to make the ci happy
```suggestion
Additionaly to its normal attributes, every unit defined by an add-on needs to define exactly one of the special attributes ``__skip_if_exists`` or ``__replace_if_exists``. This attribute determines how the engine proceeds if a description of the same name has already been registered. Units not contained in an add-on may not use these attributes.
```

Reply:
that is correct. How about adding this information after line 14 of this file (line 178 in the other file) where attributes are first mentioned.  
the complain of the ci doc check was wha i was sugesting to remove the reference.